static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -68,15 +68,15 @@ define([
          *
          * @constructor
          * @extends Component
-         * @param (Object=) $adSlot
+         * @param (Object=) adSlot
          * @param {Object=} params
          */
-        CommercialComponent = function ($adSlot, params) {
+        CommercialComponent = function (adSlot, params) {
             this.params = params || {};
             this.type   = this.params.type;
             // remove type from params
             delete this.params.type;
-            this.$adSlot    = $adSlot;
+            this.adSlot    = adSlot;
```

Comment:
I've noticed how dom elements and bonzo objects are discriminated against each other using the $ symbol for bonzo instances – a common jQuery-style naming convention.